docs/zh-CN/administrator-guide/config/be_config.md

```diff
@@ -1160,8 +1160,8 @@ storage_flood_stage_usage_percent和storage_flood_stage_left_capacity_bytes两
 
   `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,10;/home/disk2/doris`
 
-  * /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
-  * /home/disk2/doris.SSD 10， 存储限制为10GB，SSD；
+  * /home/disk1/doris.HDD,50，表示存储限制为50GB，HDD;
+  * /home/disk2/doris.SSD,10，存储限制为10GB，SSD；
```

Comment:
```suggestion
  * /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
  * /home/disk2/doris.SSD, 10，存储限制为10GB, SSD；
```

Reply:
These instructions are used to explain the configuration `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,10;/home/disk2/doris`.
The configuration is separated into three parts by the semicolon.
1. /home/disk1/doris.HDD,50
2. /home/disk2/doris.SSD,10
3. /home/disk2/doris

Using the original text is more readable. Therefore, I think my modification is better than the suggestion.